### PR TITLE
Fix typo computeBonesUsingShaders default value

### PR DESCRIPTION
**Pull request by noname0310**
the default value of `AbstractMesh.computeBonesUsingShaders` is true

https://github.com/BabylonJS/Babylon.js/blob/master/packages/dev/core/src/Meshes/abstractMesh.ts#L96

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/13965/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13965/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13965/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/13965/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/13965/merge
https://gui.babylonjs.com/?snapshot=refs/pull/13965/merge
https://nme.babylonjs.com/?snapshot=refs/pull/13965/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/13965/merge#BCU1XR#0

File: packages/dev/core/src/Meshes/abstractMesh.ts [hunk 1]
```diff
@@ -587,7 +587,7 @@ export class AbstractMesh extends TransformNode implements IDisposable, ICullabl
     }
 
     /**
-     * Gets or sets a boolean indicating that bone animations must be computed by the CPU (false by default)
+     * Gets or sets a boolean indicating that bone animations must be computed by the CPU (true by default)
```
Comment (Popov72) [inline packages/dev/core/src/Meshes/abstractMesh.ts:590]: Actually the comment was right, as it is saying "CPU".

However, I think it's clearer to say "GPU" instead, and in that case `true` is the right default value.

**Review by Popov72 — CHANGES_REQUESTED** [review]

File: packages/dev/core/src/Meshes/abstractMesh.ts (hunk 1 above)
Comment (Popov72) [reply packages/dev/core/src/Meshes/abstractMesh.ts:590]: I agree, but you should replace "CPU" by "GPU" for the doc to be accurate.

Comment (noname0310) [reply packages/dev/core/src/Meshes/abstractMesh.ts]: I applied it

**Review by sebavan — APPROVED** [review]

**Pull request merged**